plugins/templates/templates/default.js

```diff
@@ -27,61 +27,13 @@ CKEDITOR.addTemplates( 'default', {
 		title: 'Strange Template',
 		image: 'template2.gif',
 		description: 'A template that defines two columns, each one with a title, and some text.',
-		html: '<table cellspacing="0" cellpadding="0" style="width:100%" border="0">' +
-			'<tr>' +
-				'<td style="width:50%">' +
-					'<h3>Title 1</h3>' +
-				'</td>' +
-				'<td></td>' +
-				'<td style="width:50%">' +
-					'<h3>Title 2</h3>' +
-				'</td>' +
-			'</tr>' +
-			'<tr>' +
-				'<td>' +
-					'Text 1' +
-				'</td>' +
-				'<td></td>' +
-				'<td>' +
-					'Text 2' +
-				'</td>' +
-			'</tr>' +
-			'</table>' +
-			'<p>' +
-			'More text goes here.' +
-			'</p>'
+		htmlFile: CKEDITOR.getUrl( 'plugins/templates/templates/html/strange-template.html' )
```

Comment:
It's probably better to leave default templates as they were earlier. Especially because there is a difference in the behaviour: old templates are inserted synchronously and the new ones – asynchronously.

Reply:
> It's probably better to leave default templates as they were earlier. Especially because there is a difference in the behaviour: old templates are inserted synchronously and the new ones – asynchronously.

Okay. So should I come up with one or two new templates that are stored as a htmlFile? Something like title and text or whatever. As a simple example how to use htmlFile.

Reply:
AS a suggestion I know added two new template files. Is this what you meant?

Reply:
Actually I just thought about not touching this file at all. Tests should be enough to check if the new feature works.

Reply:
I thought default.js serves as an example file for developers to see how you can use this template plugin and insert new templates. Wouldn't it be better to have at least one example on how to use the htmlFile feature? I don't think the people who use this plugin would have a look at the tests to understand the feature.

Reply:
In my opinion, such an example should be added to docs, not to the code. I'd leave the file with default templates untouched as some users can be using them without any modifications in their projects and suddenly the list of available templates would become different after updating the editor.